watertap/tools/oli_api/client.py

```diff
@@ -44,31 +44,50 @@
 
 __author__ = "Adam Atia, Adi Bannady, Paul Vecchiarelli"
 
+import logging
+
+import yaml
 
 from os.path import isfile, islink
-import requests  # from requests import get, post, request
+import requests
 import json
 import time
 
 from watertap.tools.oli_api.util.watertap_to_oli_helper_functions import get_oli_name
 
+_logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter(
+    "OLIAPI - %(asctime)s - %(levelname)s - %(message)s", "%H:%M:%S"
+)
+handler.setFormatter(formatter)
+_logger.addHandler(handler)
+_logger.setLevel(logging.DEBUG)
+
 
 class OLIApi:
     """
     A class to wrap OLI Cloud API calls and access functions for interfacing with WaterTAP.
     """
 
-    def __init__(self, credential_manager, test=False):
+    def __init__(
+        self,
+        credential_manager,
+        interactive_mode=True,
+    ):
         """
         Constructs all necessary attributes for OLIApi class
 
         :param credential_manager_class: class used to manage credentials
-        :param test: bool switch for automation during tests
+        :param interactive_mode: bool to switch level of logging display from info to debug only
```

Comment:
`outlvl` would be more consistent with what we use to set logging in WaterTAP/IDAES, but not a big deal to have this deviation.

Reply:
`interactive_mode` also affects a few other lines (such as write permission) so I just left it as is for now